presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentViewModel.kt

```diff
@@ -31,6 +31,12 @@ open class AssessmentViewModel(
         assessmentNodeState.goBackward()
     }
 
+    fun reviewInstructions() {
+        val interruptionHandling = (assessmentNodeState.node as Assessment).interruptionHandling
+        assessmentNodeState.showFullInstructions = true
+        goToNode(interruptionHandling.reviewIdentifier!!)
```

Comment:
So... if the `reviewIdentifier` is null then this will crash b/c you are forcing an optional. While I realize that the code below checks for null before forcing, this wouldn't. Should either pass in the identifier or check for null.

Also, looks like this navigation will break if jumping to the review identifier from inside a branch?

Reply:
Added null check around goToNode for the reviewIdentifier in latest commit. Also, added test to confirm that wrapped assessments work.